ansible_collections/arista/avd/roles/eos_designs/templates/network_services/logic.j2

```diff
@@ -38,6 +38,21 @@
 {%                     for svi in vrf.svis | arista.avd.convert_dicts('id') | arista.avd.natural_sort('id') %}
 {%                         if svi.id | int in whitelist.vlans %}
 {%                             if "all" in match_tags or svi.tags | arista.avd.default(['all']) is arista.avd.contains(match_tags) %}
+{%                                 if svi.profile is arista.avd.defined %}
+{%                                     set svi_profile = svi_profiles | arista.avd.default({}) |
+                                                                        arista.avd.convert_dicts('profile') |
+                                                                        selectattr('profile', 'arista.avd.defined', svi.profile) |
+                                                                        first %}
+{%                                     if svi_profile.parent_profile is arista.avd.defined %}
+{%                                         set svi_parent_profile = svi_profiles | arista.avd.default({}) |
+                                                                                   arista.avd.convert_dicts('profile') |
+                                                                                   selectattr('profile', 'arista.avd.defined', svi_profile.parent_profile) |
+                                                                                   first %}
```

Comment:
Any reason to just not reuse
```
{% set parent_profile = svi_profile.parent_profile | arista.avd.default({}) %}
```

Like was done in `ip-igmp-snooping.j2` ?

Reply:
The implementation that was done in `ip-igmp-snooping.j2` was broken. Inheritance from parent_profile didn't work.
We need to handle the data model conversion. I will work on enhancing unit tests in this area.